modules/notifications/notifications.php

```diff
@@ -147,13 +147,36 @@ public static function notification_status_change( string $new_status, string $o
 
 			$action = 'status-change';
 
-			self::schedule_emails( $action, $post, $subject, $body );
+			$notification_email = OptionsUtilities::get_options_by_key( 'email_address' );
```

Comment:
@ingeniumed The notification code has been modified to check for email/webhook presence in this upper function. We use this data (`$is_email_scheduled` and `$is_webhook_scheduled`) to determine if we should trigger `vw_notification_status_change` telemetry at all, and also to include which notification types were sent in that datapoint.